installer/release.sh

```diff
@@ -58,6 +58,9 @@ do
 	--include-pdbs)
 		include_pdbs=t
 		;;
+	--include-arm64-artifacts=*)
+		arm64_artifacts_directory="${1#*=}"
```

Comment:
I just realized that due to the `cd` above, we cannot allow relative paths here. Therefore, I would like to recommend adding something like this to that clause:

```sh
case "${1#*=}" in
/*) ;; # absolute path, okay
*) die "Need an absolute path: $1";;
esac
```